ovisbot/core.py

```diff
@@ -204,3 +205,38 @@ def launch():
     if token is None:
         raise ValueError(i118n._("DISCORD_BOT_TOKEN variable has not been set!"))
     bot.run(token)
+
+@bot.command()
+async def rank(ctx, *param):
+    if len(param) > 1:
+        status_response = i118n._("Rank command takes one argument!")
+        await ctx.channel.send(status_response)
+    elif len(param) ==  0:
+        status_response = i118n._("Rank command takes one argument!")
+        await ctx.channel.send(status_response)
```

Comment:
Dame kalitera na ta valis j ta dio se 1 conditional j na sirneis exception p ginete handled se allo function  i.e.
```
@rank.error
async def rank_error(ctx):
    ....
```

Reply:
ok as ena to psa3w pos ginete